hybrid-cloud-patterns/connected-vehicle-architecture
# common automatic update

- Introduce an argo-healthcheck make target
- Adding label validatedpatterns.io/pattern to all applications.
- CI test updates
- Rework installation target
- Simplify the loop
- Introduce a validate-cluster target in the install target
- Increase the wait for the internal registry
- Add a note about SNOs and internal registries
- Move from resourceCustomization to resourceHealthcheck
- Fix up common/ tests
- Upgrade to ESO 0.9.5
- Release 0.0.3 golang-external-secrets
- Release 0.0.3 clustergroup
- Allow custom templating in .extraValueFiles
- Support pattern-wide templated value files
- Update tests
- Pass in platform and ocp version as charts would expect
- Add test for value file template expansion
- Drop the Template.{Name,BasePath} hack due to problems with the imperative configmap
- Fix up tests after last PR
- Release clustergroup v0.0.4
- Add --pull=newer when running the container
- Allow imperative to be nil
- Adding key to exclude target namespace in operatorgroup
- Added target namespace logic to namespace map case
- Changed description in schema
- Added example to operatorgroupExcludeTargetNS
- Fixing CI tests
- Push localClusterName to remote clusters too
- Preview a chart based on the current k8s cluster
- Handle explcit value files
- Update CRD for the operator
- Add a README containing the CRD update instructions
- Add ability to read overrides
- Clean up tests after 7cda9c4
- Add preview-all and remove some spurious stdout output
- All prototype preview-all and silence some output
- - Removed new key operatorgroupExludeTargetNS - Added key to namespace map entry excludeOperatorGroupTargetNS.
- Updates to CI
- Update CRD from the operator
- Make .plugin handling consistent
- Preseed the patterns-operator-config configmap
- Small IIB cleanups
- Add small curl example for IIB
- Adding option to include/exclude targetNamespaces in OperatorGroup
- Updated CI tests
- - Fix: bug in task TASK [iib_ci : Mirror all the needed images]
- - Updated the mirrordest_tag to use the sha256 of the image instead of the IIB number.
- Restored mirror template to original implementation
- - Updated structure for supporting OperatorGroup's per suggestion of decoupling operatorGroup and targetNamespaces.   Example:     - exclude-targetns:       operatorGroup: true       targetNamespaces: - Continues to support operatorgroupExcludes - Updated CI tests
- Update logic to fix multiple targetNamespaces
- Fix ci issues
- Upgraded ESO to v0.9.8
- Upgrade vault-helm to v0.26.1
- Parametrize ESO caProvider fields
- Simplify target namespace logic
- Avoid nonhubCluster + hubCluster naming for ESO
- Update for new configmanagement plugin feature
- Remove obsolete comment and update tests
- Update schema
- Require plugin.yaml
- Add tmpdir to sidecar mounts
- True up to test code
- Use nindent as appropriate
- Remove stray files
- Plugin config is plugin.yaml
- Remove now-obsolete kustomize-renderer example
- Allow pluginArgs to be set and add schema
- Remove redundancy
- Revert "Remove now-obsolete kustomize-renderer example"
- Remove legacy configManagementPlugins support
- Add configManagementPlugins to tests for industrial edge
- Clustergroup 0.0.5
- Small whitespace test
- Stop referencing remote actions via @main. Use a specific commit
- Updated ESO to v0.9.9
- Updated vault-helm to v0.27.0
- Prevent ArgoCD from writing ESO CRs to clusters that need full support
- Fix whitespaces
- Release clustergroup v0.8.0
- Document preview limitations
- Add support for private repos
- Amend tests
- Check for rc attribute to exist
- Upgrade default imperative image
- Release clustergroup v0.8.1
- Update pattern operator CRD
- Update CRD from the operator
- Bump actions/setup-python from 4 to 5
- Release clustergroup v0.8.2
- Update CRD from the operator
- Small clarification in IIB
- Switch imageDigestMirrors to AllowContactingSource
- Upgrade ESO to v0.9.10
- Add initial support for deploying private repos via CLI directly
- Update tests after common rebase
